### PR TITLE
Added second page and button to swap

### LINKED ISSUE #44: If we go with 2 pages, create HTML layout for second page, including a link back to the first page

### LINKED ISSUE #94: Separate duplicate items, like navbar, into new files

### PR DESCRIPTION
**Pull request by cmcahoon01**

### CODE REVIEW

**Comment by Davis245** [comment]
Looks good. The only thing is that I think we decided on the map being 1/3 of the width and this pull request changes the map to be 1/2 of the width. 
I think 1/3 works better because it gives us more room for the widgets; however, I think we need more comments from the group on this issue to see which width we like better as a group.

**Comment by carsondrobe** [comment]
> Looks good. The only thing is that I think we decided on the map being 1/3 of the width and this pull request changes the map to be 1/2 of the width. I think 1/3 works better because it gives us more room for the widgets; however, I think we need more comments from the group on this issue to see which width we like better as a group.

^, I think I prefer the 1/3 width map but am open to reasoning for a 1/2 view

File: Site/bc_weather_station_dashboard/website/templates/weather.html [hunk 1]
```diff
@@ -176,7 +128,12 @@ <h2 class="card-text">12</h2>
         </div>
     </div>
 
-    <!-- The Modal -->
+    <!-- Swap pages button -->
+    <a href="{% url 'fire' %}">
+        <img id="swapPageIcon" src="{%  static 'flame_icon.svg' %}" alt="fire_icon" >
```
Comment (carsondrobe) [inline Site/bc_weather_station_dashboard/website/templates/weather.html:157]: Seems to still be an issue with image sizing showing as a very large button on the bottom left of screen

File: Site/bc_weather_station_dashboard/website/views.py [hunk 2]
```diff
@@ -8,10 +8,12 @@
 from .models import WeatherStation
 
 
-# Create your views here.
 def home(request):
     return render(request, "home.html", {})
```
Comment (carsondrobe) [inline Site/bc_weather_station_dashboard/website/views.py]: home view needs to point to weather.html now

File: Site/bc_weather_station_dashboard/website/urls.py [hunk 3]
```diff
@@ -25,4 +25,6 @@
     path("login/", views.login, name="login"),
     path("weather_stations_data/", weather_stations_data, name="weather_stations_data"),
     path("submit_feedback/", views.submit_feedback, name="submit_feedback"),
+    path("weather", views.home, name="weather"),
```
Comment (carsondrobe) [inline Site/bc_weather_station_dashboard/website/urls.py]: I think you need to include a '/' after weather

**Review by carsondrobe — CHANGES_REQUESTED** [review]
I tried leaving some comments via GitHub code review. Let me know if you can't find them.

**Comment by cmcahoon01** [comment]
I changed it to 1/2 because the placeholder widgets used one half. When we design the final widgets, we can make them wider and shrink the map.

File: Site/bc_weather_station_dashboard/website/templates/weather.html (hunk 1 above)
Comment (cmcahoon01) [reply Site/bc_weather_station_dashboard/website/templates/weather.html:157]: can you try pressing ctrl + f5 to see if this goes away? or maybe cmd + f5 on mac

**Comment by robertyac** [comment]
I can't get onto the page since home doesn't exist now. I tried @Davis245 's  pull requests too and I get the same thing. Looks like the settings or urls files need to be updated.

**Comment by abijeet-dhillon** [comment]
> I can't get onto the page since home doesn't exist now. I tried @Davis245 's pull requests too and I get the same thing. Looks like the settings or urls files need to be updated.

Having the same issue as well.

**Comment by cmcahoon01** [comment]
I believe I have fixed these issues, please let me know if it works now

File: Site/bc_weather_station_dashboard/website/templates/fire.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+<body>
+
+    <!-- NAVBAR -->
+    {% include 'navbar.html' %}
+
+    <div class="container-fluid">
+        <div class="row pt-3">
+            <!-- Station Map Column -->
+            <div class="col-6 text-center border-right">
```
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/templates/fire.html:34]: I personally like col-4 better for the map, the slimmer map to give more room for the widgets.

Comment (robertyac) [reply Site/bc_weather_station_dashboard/website/templates/fire.html:34]: But its up to whatever everyone thinks...

**Review by robertyac — APPROVED** [review]
LGTM as long as the current home temp widget does't disappear and just gets overridden like you said.

**Comment by cmcahoon01** [comment]
> LGTM as long as the current home temp widget does't disappear and just gets overridden like you said.

It is still there:
Animation2

**Comment by robertyac** [comment]
> > LGTM as long as the current home temp widget does't disappear and just gets overridden like you said.
> 
> It is still there: Animation2 [ Animation2 ](https://private-user-images.githubusercontent.com/31668119/311079595-4056ffb3-5f77-4786-b865-40d6b7277f68.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4NTg3NDgsIm5iZiI6MTcwOTg1ODQ0OCwicGF0aCI6Ii8zMTY2ODExOS8zMTEwNzk1OTUtNDA1NmZmYjMtNWY3Ny00Nzg2LWI4NjUtNDBkNmI3Mjc3ZjY4LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA4VDAwNDA0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjMmI5MjkwYjczZGY5ZDhkODUzODA0YTVlMjc5MjNjYzJhMDYxZjY3ZTE5NTRhNTY4YTk5YjJjNmI5N2EyMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._Y5NIQl41r-v0m93LBN1-SsmWUA835OWQjtdhWjA2PI) [ ](https://private-user-images.githubusercontent.com/31668119/311079595-4056ffb3-5f77-4786-b865-40d6b7277f68.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4NTg3NDgsIm5iZiI6MTcwOTg1ODQ0OCwicGF0aCI6Ii8zMTY2ODExOS8zMTEwNzk1OTUtNDA1NmZmYjMtNWY3Ny00Nzg2LWI4NjUtNDBkNmI3Mjc3ZjY4LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA4VDAwNDA0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjMmI5MjkwYjczZGY5ZDhkODUzODA0YTVlMjc5MjNjYzJhMDYxZjY3ZTE5NTRhNTY4YTk5YjJjNmI5N2EyMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._Y5NIQl41r-v0m93LBN1-SsmWUA835OWQjtdhWjA2PI)

Weird I refreshed my cache too and it isn't showing 

Screenshot 2024-03-07 at 4 42 09 PM

**Comment by cmcahoon01** [comment]
Did you pull since that merge?

**Comment by robertyac** [comment]
Yeah I thought I did... apparently not lol looks good

**Review by carsondrobe — APPROVED** [review]
Works as expected! LGTM

**Comment by carsondrobe** [comment]
Just need to resolve conflicts

**Pull request merged**